pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOK_CEPH_MON_HOST="{{ .RookCephMonHost }}"
+ROOK_CEPH_MON_INITIAL_MEMBERS="{{ .RookCephMonInitialMembers }}"
+DAEMON_MDS="{{ .DaemonMds }}"
+FILESYSTEM_NAME="{{ .FilesystemName }}"
+
+outp="$(ceph fs dump --mon-host="$ROOK_CEPH_MON_HOST" --mon-initial-members="$ROOK_CEPH_MON_INITIAL_MEMBERS" --format json 2>&1)"
+rc=$?
+if [ $rc -ne 0 ]; then
+    echo "ceph mds dump check failed with the following output:"
+    echo "$outp" | sed -e 's/^/> /g'
+    echo "we couldn't get the information to reliably tell if the MDS should be restarted or not"
+    return 0
```

Comment:
```suggestion
    exit 0
```

Reply:
can we clear on this, 

So to return a error -> we do `exit 0` and just to return to the end of function we use `return 0` ?